pages/_app.js

```diff
@@ -3,6 +3,9 @@ import Head from 'next/head';
 import configureNProgress from 'config/nprogress';
 import { AppProvider } from 'providers';
 
+import 'stream-chat-react/dist/css/index.css';
+import './StreamChatOverrides.css';
```

Comment:
I believe this means this CSS will be included on every page? Perhaps we can use the `<Head>` component from next to add a stylesheet only on the pages where this is needed?

Thoughts?

Reply:
So I tried to have the imports only in `components/Chat/Chat.js` at first, figuring only pages using `<Chat>` would need it. But Next.js gives this error, which I didn't try _too_ hard to work around since they gave a specific recommendation — but this is a great point.

I can dig a bit further, but any thoughts?
Encountered this before?

<img src="https://user-images.githubusercontent.com/1812955/112575415-a9fe7a00-8dc6-11eb-93e9-b71ac10fb9cd.png" width="600" />

```
error - ./components/Chat/StreamChatOverrides.css
Global CSS cannot be imported from files other than your Custom <App>. Please move all global CSS imports to pages/_app.js. Or convert the import to Component-Level CSS (CSS Modules).
Read more: https://err.sh/next.js/css-global
Location: components/Chat/Chat.js
```